docs/modules/plugins/pages/plugin-kafka.adoc

```diff
@@ -10,6 +10,27 @@ The plugin provides integration with an open-source distributed event streaming
 implementation(group: 'org.vividus', name: 'vividus-plugin-kafka', version: '{current-version}')
 ----
 
+== Deprecations
+
+=== Steps
+
+Following list of steps are deprecated and will be removed in the following releases:
```

Comment:
`The following list of steps is deprecated and will be removed in the future releases:`